src/views/Game/AIReview.tsx

```diff
@@ -687,6 +709,220 @@ export class AIReview extends React.Component<AIReviewProperties, AIReviewState>
         return false;
     }
 
+    private getPlayerColorsMoveList(){
+        const init_move = this.props.game.goban.engine.move_tree;
+        const move_list = [];
+        let cur_move = init_move.trunk_next;
+
+        while (cur_move !== undefined){
+            move_list.push(cur_move.player);
+            cur_move = cur_move.trunk_next;
+        }
+        return move_list;
+
+    }
+
+    private AiSummaryTableRowList(){
+        const summary_moves_list = [ ["", "", "", ""], ["", "", "", ""], ["", "", "", ""], ["", "", "", ""], ["", "", "", ""], ["", "", "", ""] ];
+        const ai_table_rows = [["Excellent"], ["Great"], ["Good"], ["Inaccuracy"], ["Mistake"], ["Blunder"]];
```

Comment:
Would be good to internationalize these strings:

```
[[_("Excellent")], [_("Great")], [_("Good")], [_("Inaccuracy")], [_("Mistake")], [_("Blunder")]]
```

Reply:
Sure, I assume that so they can be translated? Makes senes, I didn't know the syntax :)

Reply:
The two main ones are _("String") and [pgettext](https://github.com/online-go/online-go.com/blob/13fcc8b1e698eebb80b90790d6e8fe522c35dc8f/src/lib/translate.ts#L173) which allows you to add additional context for more complex strings

Reply:
Is there a simple way to just click to accept the suggested change?

Reply:
```suggestion
        const ai_table_rows = [[_("Excellent")], [_("Great")], [_("Good")], [_("Inaccuracy")], [_("Mistake")], [_("Blunder")]];
```

Reply:
Thanks for making that change!  Yeah basically there's a script that picks up all the _(...) and pgettext(...) and sends them to translate.online-go.com for translation.  Once the translators do their thing, the translations get added to the dictionary and these functions actually convert the text to the target language.

Pro tip: if you change your language to *Debug* in the settings, it shows you which strings have already been internationalized.  Strings wrapped in [...] are ready:

<img width="380" alt="Screen Shot 2021-12-16 at 9 24 19 AM" src="https://user-images.githubusercontent.com/25233703/146389751-b3f36da6-831d-46f5-9b09-6a6942a1baf8.png">

Reply:
That sounds like a really convenient idea, to have a debug language like that :)